Declare RunApplicationEventLoop and QuitApplicationEventLoop to fix:
libs/ui/ui.c:197:2: error: implicit declaration of function 'RunApplicationEventLoop' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
libs/ui/ui.c:217:2: error: implicit declaration of function 'QuitApplicationEventLoop' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/HaxeFoundation/neko/commit/1fa630f7db21fe5edec23ed4734b9d47e40a069d
--- libs/ui/ui.c.orig
+++ libs/ui/ui.c
@@ -36,6 +36,8 @@
 #	define UIEvent		0xFFFFAA00
 #	define eCall		0x0
 enum { pFunc = 'func' };
+extern void RunApplicationEventLoop(void);
+extern void QuitApplicationEventLoop(void);
 #else
 #	include <gtk/gtk.h>
 #	include <glib.h>
